src/components/Operations.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class Operations extends Component {
+
+    constructor(props){
+        super(props);
+        this.state= {
+            amount : 0,
+            category : '',
+            vendor : ''
+        }
+    }
+
+    amountChange = event => {
```

Comment:
make an input handler and handles all inputs. You do not need one for each value. 
    handleInput= event => {
        this.setState({ [event.target.name]: event.target.value });
      }